### PR TITLE
add click-config-file

### PR DESCRIPTION
**Pull request by ltalirz**
Checklist

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages
- [x] Build number is 0
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by ltalirz** [comment]
contacted package maintainer @phha; will add him as a second maintainer (or not) when he's finished considering
https://github.com/phha/click_config_file/pull/2#issuecomment-484464724

File: recipes/click_config_file/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 12bc380aa39734749a070b498a21d8d0b2314e38f9320b592fe213a316be5354
+
+build:
+  number: 0
```
Comment (chrisburr) [inline recipes/click_config_file/meta.yaml]: This looks like it could be a noarch package?
```suggestion
  number: 0
  noarch: python
```

**Comment by ltalirz** [comment]
@chrisburr thanks a lot for the swift action, much appreciated!

**Comment by chrisburr** [comment]
I don’t have the rights to do it but we can ping the core team and you can ask in the gitter chat if needed.

@conda-forge/core is it practical to rename a package that was just merged or delete it so it can be re-added?

**Comment by ocefpaf** [comment]
> @conda-forge/core is it practical to rename a package that was just merged or delete it so it can be re-added?

Just send a PR to the feedstock with the new name. We can move the old name to broken. No need to rename the feedstock in this case.

**Comment by ltalirz** [comment]
@ocefpaf Thanks! Here is the PR: https://github.com/conda-forge/click_config_file-feedstock/pull/1

**Comment by ltalirz** [comment]
@ocefpaf I interpreted the thumbs-up such that the PR was good to merge (?), so I merged it.
The build after the merge has completed but `click-config-file` does not yet appear in the `conda-forge` channel.
Should I have waited with merging the PR? I can easily make a new one with a new build number if that's the case.

**Comment by chrisburr** [comment]
@ltalirz It can take up to an hour for new/updated packages to appear on the CDN. The file has been uploaded so it should work soon: https://anaconda.org/conda-forge/click-config-file/files

**Comment by ltalirz** [comment]
Ah, thanks! It's there now :-)

**Pull request merged**